### PR TITLE
added support for svg download from assets [Fixes #8026]

### LINKED ISSUE #8026: Feature request: add ETH assets in SVG
### Is your feature request related to a problem? Please describe.

There are many reasons why someone might need our assets in SVG (especially the glyph logo section)

Current issues:

1. Our website does not provide svg format on many of the brand assets
2. If there is SVG version on the website, there is no download button

### Describe the solution you'd like

1. provide an option to download ethereum glyph in svg format
2. add download button for svg assets

### Describe alternatives you've considered

some users also requested EPS format, but SVG would be enough.

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by kaspi90**

### CODE REVIEW

**Review by Inatiye33 — APPROVED** [review]

File: src/components/AssetDownload.tsx [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
+import * as ReactDOMServer from "react-dom/server"
+import Button from "./Button"
 import Translation from "../components/Translation"
 import ButtonLink from "./ButtonLink"
 import Emoji from "./OldEmoji"
 import Link from "./Link"
 
 import { getImage, getSrc, ImageDataLike } from "../utils/image"
+import { Box, Flex, Wrap, WrapItem } from "@chakra-ui/react"
+import theme from "../@chakra-ui/gatsby-plugin/theme"
```
Comment (pettinarip) [inline src/components/AssetDownload.tsx:14]: Some of these imports aren't used. Lets remove them.

And external lib imports should be imported at the top of the file (along with the react, react-dom import).

File: src/components/AssetDownload.tsx [hunk 2]
```diff
@@ -100,6 +106,40 @@ const AssetDownload: React.FC<IProps> = ({
...
+  const [svgUrl, setSvgUrl] = useState<string>("")
+  const SvgImage = () => {
+    return <>{Svg && <Svg alt={alt} />}</>
+  }
+
+  const getSvgRendered = () => {
+    const svgRendered = ReactDOMServer.renderToString(<SvgImage></SvgImage>)
+
+    const addingSizeText = 'height="64" width="64" '
+    const index = svgRendered.indexOf("<svg ") + 5
```
Comment (pettinarip) [inline src/components/AssetDownload.tsx:118]: Curious to understand why we need that extra `5` in this index.

**Review by pettinarip — CHANGES_REQUESTED** [review]
Thanks for the PR @kaspi90 

This svg doesn't have a download button. Wondering if we need to add it there as well.
cc @konopkja 
image

When I downloaded the "ETH diamond (glyph)" and the "ETH diamond (color)" svgs, I got this error when I tried to open them on the browser:
image
Looking at the svg code it seems that we are injecting an `<image id="_Image1" width="64">` tag inside it. I think that is not what we want. We should have `<path>`s instead.

**Comment by konopkja** [comment]
> 

i think merge related artwork was supposed to be temporary? what do you think @nloureiro

**Comment by corwintines** [comment]
2 things to follow up with from grooming:
- We will move the merge images into the historical section of this page
- Instead of png and svg each having their own image, we will instead just show 1 image and have a button for each image type

**Comment by pettinarip** [comment]
@konopkja this is ready for review. [Preview](https://ethereumorgwebsitedev01-kaspi90ethereumorgwebsitesvgas.gatsbyjs.io/en/assets/)

**Comment by konopkja** [comment]
@pettinarip  buttons look good

Question: can we make something about the preview picture sizes? :D 

Screenshot 2023-05-15 at 20 44 49

**Comment by nhsz** [comment]
> Preview

checking previous comments, should merge images move to the historial section? @pettinarip @corwintines

**Comment by nhsz** [comment]
> 2 things to follow up with from grooming:
> 
> * We will move the merge images into the historical section of this page
> * Instead of png and svg each having their own image, we will instead just show 1 image and have a button for each image type

checking previous comments, should merge images move to the historial section? @pettinarip @corwintines

**Comment by pettinarip** [comment]
Goood point @nhsz. I'll refresh my mind with those requirements and will get back to you.

**Comment by pettinarip** [comment]
Done @konopkja @nhsz. I've moved the panda images under the historical images section.

**Review by corwintines — APPROVED** [review]

**Pull request merged**